pkg/component/observability/monitoring/prometheus/garden/prometheusrules.go

```diff
@@ -196,6 +196,29 @@ func gardenPrometheusRule(isGardenerDiscoveryServerEnabled bool) *monitoringv1.P
 					"last hour. The pod is running on the garden cluster.",
 			},
 		},
+		{
+			Alert:  "GardenConditionNotTrue",
```

Comment:
Maybe let's call this alert `GardenConditionStatusNotTrue` to not confuse this with the condition reason.